helm/kvm-operator/templates/configmap.yaml

```diff
@@ -14,6 +14,8 @@ data:
     service:
       crd:
         labelSelector: '{{ .Values.Installation.V1.GiantSwarm.KVMOperator.CRD.LabelSelector }}'
+      podSecurityPolicy:
+        name: {{ tpl .Values.resource.psp.name . }}
```

Comment:
Not sure if this needs to be quoted like `labelSelector` above.

Reply:
I think this is always going to be a valid YAML string so it's not necessary to quote.